src/libAtomVM/resources.c

```diff
@@ -102,5 +102,7 @@ ERL_NIF_TERM enif_make_resource(ErlNifEnv *env, void *obj)
     if (UNLIKELY(memory_erl_nif_env_ensure_free(env, TERM_BOXED_RESOURCE_SIZE) != MEMORY_GC_OK)) {
         AVM_ABORT();
     }
+    struct RefcBinary *refc = refc_binary_from_data(obj);
+    refc_binary_increment_refcount(refc);
```

Comment:
Just out of curiosity, is this fixing a bug that we missed in the earlier commit?  I am fine with the change -- just wondering why.

Reply:
I changed the semantics of term_from_resource after @bettio ´s comment.

enif_make_resource must be followed by an enif_release_resource as [documented](https://www.erlang.org/doc/man/erl_nif.html#enif_make_resource).

term_from_resource must not.

I just moved the refc increment from term_from_resource to enif_make_resource.